hipogram/posts/templates/post_list.html

```diff
@@ -2,18 +2,28 @@
 
 {% block content %}
 
+  {% if messages %}
+  <ul class="messages">
+      {% for message in messages %}
+      <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
+      {% endfor %}
+  </ul>
+  {% endif %}
+
   <div class="container">
     {% for post in posts %}
     <aside>
       <p>Trend Tags</p>
     </aside>  
       <div class="card mb-3">
         <div class="card-body">
-          <a href="{% url "posts:update_post" %}">Update Post</a></br>
-          <a href="{% url "posts:delete_post" %}">Delete Post</a>
-          <h2 class="card-title">{{ post.created_by.username }}</h2>
+          <h2 class="card-title"><a href="{{ post.created_by.username }}">{{ post.created_by.username }}</a></h2>
```

Comment:
This gives error since we don't have any url like `{{ post.created_by.username }}`, so it should be in this PR. Things we added for testing shouldn't be commited.